src/extras/curves/ClosedSplineCurve3.js

```diff
@@ -3,37 +3,13 @@
  **************************************************************/
 
 
-THREE.ClosedSplineCurve3 = THREE.Curve.create(
+THREE.ClosedSplineCurve3 = function ( points ) {
 
-	function ( points /* array of Vector3 */ ) {
+	console.warn( 'THREE.ClosedSplineCurve3 may be depreciated. Please consider THREE.CatmullRomCurve3' );
```

Comment:
Your welcome, but I think you mean _deprecated_, not _depreciated_. (2 instances)